package.json

```diff
@@ -5,8 +5,10 @@
   "scripts": {
     "start": "docusaurus start",
     "build": "docusaurus build",
-    "lint": "npm run lint:links && npm run lint:spelling && npm run lint:style",
-    "lint:links": "remark -f -q -u validate-links \"docs/**/*.mdx\"",
+    "lint": "npm run lint:spelling && npm run lint:links && npm run lint:style",
+    "lint:links": "npm run lint:links:md && npm run lint:links:ext",
+    "lint:links:md": "remark -f -q -u validate-links \"docs/**/*.mdx\"",
+    "lint:links:ext": "npm run build && linkinator ./build -r --silent --concurrency 10",
```

Comment:
I don't think we should require `build` to have been run here. It doubles our build time which is around ~50s in CI.

Reply:
If we don't require build to be run, then how can we be sure we are checking the latest up to date website's external links?

Reply:
`build` is run before `lint` in CI. Is that what you mean?

Reply:
doh, my bad. I was thinking local. Of course build runs before lint in CI.

Removed the `npm run build ` call from the lint step.